### DISCUSSION TITLE
Response from AI?

### DISCUSSION THREAD

**Discussion by waldis007** [danielmiessler/Fabric#484] [Q&A]
I'm trying to set it up and all I'm getting is unformatted transcript from YT, and the following from gpt-4o:
```
# SUMMARY

# IDEAS:

# INSIGHTS:

# QUOTES:

# HABITS:

# FACTS:

# REFERENCES:

# ONE-SENTENCE TAKEAWAY:

# RECOMMENDATIONS:[1]+  Done                    yt --transcript https://www.youtube.com/watch?v=aMzdeZ8vGXQ
```

It does reach out to GPT, since it keeps deducting from my balance every time I run it with no results.

The prompt I'm testing with is as follows:

`yt --transcript https://www.youtube.com/watch?v=aMzdeZ8vGXQ&ab_channel=NetworkChuck | fabric -sp extract_wisdom`
or
`yt --transcript https://www.youtube.com/watch?v=aMzdeZ8vGXQ&ab_channel=NetworkChuck | fabric --stream --pattern extract_wisdom`

What am I doing wrong?

Thanks.

**Comment by waldis007** [comment]
Okay, a bright light bulb lit up while I was looking at what I wrote... the YT string appended with & at the end of the URL must have been messing everything up. Tried it as follows and everything worked just fine.

`yt --transcript https://www.youtube.com/watch?v=aMzdeZ8vGXQ | fabric --stream --pattern extract_wisdom`

**Reply by carlleonard** [reply]
Try surrounding the full URL in single quotes.
